fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/AbstractPhysicalPlan.java

```diff
@@ -64,6 +64,11 @@ public List<Slot> getOutput() {
         return logicalProperties.getOutput();
     }
 
+    @Override
```

Comment:
move it to `AbstractPlan`, BTW, move getOutput to `AbstractPlan`

Reply:
done